apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -661,13 +673,17 @@ function initializeBlocklyWrapper(blocklyInstance) {
     const trashcan = new CdoTrashcan(workspace);
     trashcan.init();
 
-    if (options.useModalFunctionEditor) {
-      // Customize auto-populated Functions toolbox category.
-      workspace.registerToolboxCategoryCallback(
-        'PROCEDURE',
-        functionsFlyoutCategory
-      );
-    }
+    // Customize auto-populated Functions toolbox category.
+    workspace.registerToolboxCategoryCallback(
```

Comment:
conflict here too: should this logic be here or in [functionEditor](https://github.com/code-dot-org/code-dot-org/pull/53114/files#diff-d18875c93b715dba98c6ac0cb7171f9ae07679b1859985ac703a738d8e61625aR86)?